components/automate-ui/src/app/entities/infra-roles/infra-role.action.ts

```diff
@@ -14,7 +14,10 @@ export enum RoleActionTypes {
   GET_ALL_FAILURE = 'ROLES::GET_ALL::FAILURE',
   GET = 'ROLES::GET',
   GET_SUCCESS = 'ROLES::GET::SUCCESS',
-  GET_FAILURE = 'ROLES::GET::FAILURE'
+  GET_FAILURE = 'ROLES::GET::FAILURE',
+  UPDATE            = 'ROLES::UPDATE',
+  UPDATE_SUCCESS    = 'ROLES::UPDATE::SUCCESS',
+  UPDATE_FAILURE    = 'ROLES::UPDATE::FAILURE'
```

Comment:
[ nit ] - again not your fault, while you're here though, it's chef convention to space these so that all the `=` signs fall in the same location.

an example to look at would be:
https://github.com/chef/automate/blob/2c8150322913a52ae6f72893952aabde8e790ad0/components/automate-ui/src/app/entities/teams/team.actions.ts#L6-L31